pronotepy/dataClasses.py

```diff
@@ -1991,3 +1993,66 @@ def __init__(self, client: ClientBase, json_dict: dict) -> None:
         self.duration: Optional[datetime.timedelta] = self._resolver(
             lambda v: datetime.timedelta(minutes=int(v)), "duree", strict=False
         )
+
+
+class TeachingSubject(Object):
```

Comment:
Maybe make this an inner class of `TeachingStaff`? I don't think it has any use outside this specific context.